src/compat/e-129/ArkoudaBigIntCompat.chpl

```diff
@@ -1,5 +1,9 @@
 module ArkoudaBigIntCompat {
-  use BigInteger;
+  public use BigInteger;
+
+  inline operator bigint.:(src: bool, type toType: bigint): bigint throws {
```

Comment:
Looks good, but can i just get an explantation of what the `operator` part of this `inline` function does.

Reply:
yah, it's worth noting i grabbed this snippet from @bmcdonald3 in the chapel issue, but i can try and explain it.

this is just defining an operator in this case `:`. It allows us to apply `:` on a `bool` and the type of `bigint`. So now when the code has something like `myBool: bigint`, it's able to find this operator and call this function with `src=myBool` and `toType=bigint`. Does that make sense?

I believe all operators in chapel are defined this way and so you would see similar something similar for other more conventional binary operations like `+`

Reply:
Thank you :)